### PR TITLE
Fix issue #208

### PR DESCRIPTION
**Pull request by mattam82**
Remove 8.7.dev packages for coquelicot and ssreflect. The .dev packages with the correct dependencies stay. Fixes issue #208

### CODE REVIEW

File: extra-dev/packages/coq-template-coq/coq-template-coq.8.7.dev/descr [hunk 1]
```diff
@@ -0,0 +1,17 @@
+A quoting and unquoting library for Coq in Coq.
```
Comment (CohenCyril) [inline extra-dev/packages/coq-template-coq/coq-template-coq.8.7.dev/descr]: This has nothing to do with #208 ...

Comment (mattam82) [reply extra-dev/packages/coq-template-coq/coq-template-coq.8.7.dev/descr]: Yeah, I just pushed over another PR, which I'll expect will be merged before that one

**Comment by CohenCyril** [comment]
Oh, isn't it a bit dirty?

**Comment by mattam82** [comment]
Well, you know how I am ;)

**Comment by maximedenes** [comment]
> Oh, isn't it a bit dirty?

It's common to do so when the new patch relies on the previous, not-yet-merged one. If the changes are orthogonal, it's a bad practice.

**Comment by mattam82** [comment]
I didn't realize I was doing that actually :)

**Pull request merged**